packages/synapse-react-client/src/components/EntityViewScopeEditor/EntityViewScopeEditor.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useMemo } from 'react'
+import EntityHeaderTable from '../EntityHeaderTable'
+import { EntityType, ReferenceList } from '@sage-bionetworks/synapse-types'
+import { VersionSelectionType } from '../EntityFinder/VersionSelectionType'
+import { FinderScope } from '../EntityFinder/tree/EntityTree'
+import { EntityTypeGroup } from '../../utils/functions/EntityTypeUtils'
+import { Typography } from '@mui/material'
+
+const DEFAULT_SELECTABLE_TYPES = EntityTypeGroup.CONTAINER
+const PROJECT_VIEW_SELECTABLE_TYPES = [EntityType.PROJECT]
+
+export type EntityViewScopeEditorProps = {
+  scopeIds: string[]
+  onChange: (scopeIds: string[]) => void
+  isProjectView?: boolean
+}
+
+export default function EntityViewScopeEditor(
+  props: EntityViewScopeEditorProps,
+) {
+  const { scopeIds, isProjectView = false } = props
+  const references: ReferenceList = useMemo(
+    () => scopeIds.map(scope => ({ targetId: scope })),
+    [scopeIds],
+  )
+
+  return (
+    <>
+      <Typography variant={'body1'} mb={1.25} sx={{ fontWeight: 700 }}>
+        Scope
+      </Typography>
+      {scopeIds.length === 0 && (
+        <Typography variant={'smallText1'} color={'grey.600'}>
+          Empty! Add items to populate your view
+        </Typography>
+      )}
+      <EntityHeaderTable
+        references={references}
+        isEditable={true}
+        onUpdate={newReferences => {
+          props.onChange(newReferences.map(ref => ref.targetId))
+        }}
+        removeSelectedRowsButtonText={'Remove Selected Items from View Scope'}
+        objectNameCopy={'container'}
```

Comment:
Not sure what the design specifies, but would it make sense to toggle `objectNameCopy` to 'projects' when `isProjectView` is true?

Reply:
Great idea!